### PR TITLE
Fixed multiple layout issues and Kitkat 4.4 minimum SDK version issue

### PR DESCRIPTION
**Pull request by profjb58**
- Filters dialog now looks acceptable on Lollipop (Android 5)
- Updated minimum SDK version to lollipop (5.0) to fix KitKat issues. See github issue #72
- Fixed portrait orientation for Presentation screen, & recipe info. Landscape orientation still works for profile settings and public profile.
- Reduced Picasso version to support older devices.

Fixes issues #72 #69 #60

### CODE REVIEW

**Comment by profjb58** [comment]
New design for 5.0 looks like the following... Kinda funky but also kinda weirdly works because of the purple, green colourscheme. Looks better on newer devices.

image

**Comment by profjb58** [comment]
[KitKat 4.4 (Multiple issues)](https://trello.com/c/pAFOeaSY/73-kitkat-44-multiple-issues)

**Comment by profjb58** [comment]
[Landscape screen rotation issue](https://trello.com/c/qaEEKG6L/72-landscape-screen-rotation-issue)

**Comment by profjb58** [comment]
[Queries - Layout issues](https://trello.com/c/msSCzbQA/64-queries-layout-issues)

File: app/build.gradle [hunk 1]
```diff
@@ -70,9 +69,10 @@ testImplementation 'junit:junit:4.12'
     implementation 'com.google.android.gms:play-services-gcm:17.0.0'
     implementation 'com.squareup.picasso:picasso:2.71828'
     implementation 'com.google.firebase:firebase-firestore:21.0.0'
+    implementation 'com.squareup.okhttp3:okhttp:3.12.2'
```
Comment (nathanbillis) [inline app/build.gradle:72]: Does this need to remain as 3.12.2. or can it go back up to 4.4.0?

**Pull request merged**